pkg/fargate/client_test.go

```diff
@@ -192,13 +205,33 @@ func mockCreateFargateProfile(mockClient *mocks.EKSAPI) {
 		Return(&eks.CreateFargateProfileOutput{}, nil)
 }
 
+func mockCreateFargateProfileWithoutTag(mockClient *mocks.EKSAPI) {
+	mockClient.Mock.On("CreateFargateProfile", testCreateFargateProfileInputWithoutTag()).
+		Return(&eks.CreateFargateProfileOutput{}, nil)
+}
+
 func mockForFailureOnCreateFargateProfile() *mocks.EKSAPI {
 	mockClient := mocks.EKSAPI{}
 	mockClient.Mock.On("CreateFargateProfile", testCreateFargateProfileInput()).
 		Return(nil, errors.New("the Internet broke down!"))
 	return &mockClient
 }
 
+func testFargateProfileWithoutTag() *api.FargateProfile {
```

Comment:
By convention we call the functions that actually perform a test with assertions, etc `test...` since this function is creating a fargate profile how about calling it `createProfileWithoutTag()`?